### PR TITLE
fix(sql): increase batch size for SqlQueue::containsMessage

### PR DESCRIPTION
**Pull request by dreynaud**
It seems that a batchSize of 100 is small enough to allow the zombie check in
orca to flag wrong things.

### CODE REVIEW

File: keiko-sql/src/main/kotlin/com/netflix/spinnaker/q/sql/SqlQueue.kt [hunk 1]
```diff
@@ -166,7 +166,7 @@ class SqlQueue(
   }
 
   override fun containsMessage(predicate: (Message) -> Boolean): Boolean {
-    val batchSize = 100
+    val batchSize = 2000
```
Comment (asher) [inline keiko-sql/src/main/kotlin/com/netflix/spinnaker/q/sql/SqlQueue.kt]: Why is this relevant to orca's zombie detector being busted?

Comment (dreynaud) [reply keiko-sql/src/main/kotlin/com/netflix/spinnaker/q/sql/SqlQueue.kt]: The zombie detector sometimes flags executions as zombies erroneously, I believe that happens because it fails to find messages on the queue for these executions between 2 batches of reads

Comment (robzienert) [reply keiko-sql/src/main/kotlin/com/netflix/spinnaker/q/sql/SqlQueue.kt]: It's worth experimenting with, but if we're going to experiment with this value, we should expose it through configuration, rather than making binary changes - we don't know how such a dramatic change will affect things (or if this number is the sweet-spot), and I'd hate to cut releases to Keiko while experimenting.

Comment (asher) [reply keiko-sql/src/main/kotlin/com/netflix/spinnaker/q/sql/SqlQueue.kt]: @dreynaud that's a plausible explanation but this PR doesn't fix the issue. It might make it less likely, but if we want to trust a zombie detector well enough to automate actions based on it, this PR may give a false sense of security.

Also note, the current zombie detector wants to load and deserialize every message on the keiko-sql messages table for every older running execution in orca. That is very inefficient and very slow. The batch size at which the raw data is fetched from the db is a marginal contributor to the time cost here, and that needs to be addressed. 

I'd like to see a zombie detector for orca+keiko-sql that:

1) selects older running executions and the keiko messages table in a single atomic operation.
2) create a `Set<String>` of execution id's on the queue in a single pass.
3) identify executions not in that set.

This could run against an in-region reader if the cost warrants it, but the point is to write a zombie detector that we can expect to be reliable as a starting point.

Comment (dreynaud) [reply keiko-sql/src/main/kotlin/com/netflix/spinnaker/q/sql/SqlQueue.kt]: Good point about this being in a loop. I'll carve some time to make this more better
